crates/sncast/tests/helpers/fixtures.rs

```diff
@@ -1,9 +1,9 @@
 use crate::helpers::constants::{ACCOUNT_FILE_PATH, CONTRACTS_DIR, DEVNET_ENV_FILE, URL};
```

Comment:
This file contains two `cast` string, but I'm unable to suggest change (although this is a test file, we should probably be consistent)

Reply:
These are not in the context of the tool but type casting